__tests__/app.test.js

```diff
@@ -153,6 +153,110 @@ describe("/api/articles/:article_id", () => {
         });
     });
   });
+  describe("PATCH", () => {
+    it("200: increments the votes property on an article by 1", () => {
+      const input_article_id = 1;
+      const newVote = 1;
+      const patchInfo = { inc_votes: newVote };
+      const expectedReturnArticle = {
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: "2020-07-09T20:11:00.000Z",
+        votes: 101,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      };
+      return request(app)
+        .patch(`/api/articles/${input_article_id}`)
+        .send(patchInfo)
+        .expect(200)
+        .then(({ body: { article } }) => {
+          expect(article).toMatchObject(expectedReturnArticle);
+        });
+    });
+    it("200: decerements the votes property on an article by 100", () => {
```

Comment:
I would remove the specifics of 'by 100' here, and instead just say that it can decrement the votes property by a given number :)